.travis.yml

```diff
@@ -3,11 +3,13 @@ os:
     - linux
     - osx
 julia:
-    - 0.6
+    - 0.7
```

Comment:
You can drop it and require at least Julia v1.0 as Polyhedra v0.5 requires Julia v1.0